solutions/promises_async_await/4.js

```diff
@@ -0,0 +1,19 @@
+const firstPromise = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    resolve("1. Finished");
+  }, 1000);
+});
+const secondPromise = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    resolve("2. Finished");
+  }, 2000);
+});
+const thirdPromise = new Promise((resolve, reject) => {
+  setTimeout(() => {
+    resolve("3. Finished");
+  }, 3000);
+});
```

Comment:
You can create a single function that takes in the seconds as argument and returns a promise